src/pages/my-hacktober-experience/hacktober.txt

```diff
@@ -0,0 +1,52 @@
+title:-
+MY EXPERIENCE WITH THE HACKTOBERFEST.
+date:"2019-10-28"
+time: "01:40:00"
+
+I was not even a noob in the field of open source
+before I was introduced with git and github in 
+the weekly meetup of our community .
+It was a hands-on session for the basic commands
+of git and we were given a brief description about
+the open source community.
+By the end of the session, it was announced that  next
+week's agenda will be 'HACKTOBERFEST'. 
+I was quite excited to run git commands on my terminal
+because as lame as it may sound, but typing anything
+on the command line is still the second best feeling 
+for me, getting a pull request merged still remains the
+first. After being familiar with the basic commands and
+principles , I started reading blogs related to hackoberfest
+and other opensource programs.
+At first , it appeared very diffcult , things were not very 
+clear and I was quite skeptical if I can pull this off.
+I am a decent programmer and I was very uncertain if I would
+be able to make even one sensible contribution.
+Then I consulted fellow community members and they gave me 
+some meaningful insights and busted all the myths .
+Communication therefore is the first and foremost important
+thing in this community.
+Having clarified all my doubts, I registered for the hacktoberfest . 
+The website was really cool and it had all the necessary information
+regarding the basic guidelines about the fest. 
+There was even a dedicated section which listed beginner friendly projects .
+I quickly went through the list and found some easy projects then and
+there only . I started working on the projects and finsihed them 
+as quickly as I could. Having made four PRs , I was quite content 
+and satisfied but I never knew that it's not as easy as it seems.
+In the greed of completing the challenge
+and claiming my t-shirt , I had made one useless PR and it was 
+rejected as it was against the community standards .
+I was shattered but I didn't give up and this time.
+I began searching for some useful yet beginner friendly projects .
+Finally , I completed the challenge and all of my PR's were accepted.
+The feeling was euphoric.
+I recieved a mail on 23rd October to claim my T-shirt and swags.
+"This part of my life, this little part is called happiness" (read in Will Smith's voice).
```

Comment:
Try enclosing this quote in a quote block. Take a look here -https://www.markdownguide.org/basic-syntax#blockquotes-1

Try giving a read about Markdown. Maybe here - https://www.markdownguide.org/getting-started or any other resource you find. It will help you to add some headings, bullets etc. to the post.